src/dashboard/src/components/header/Header.tsx

```diff
@@ -8,6 +8,7 @@ import Link from 'next/link';
 import { redirect, usePathname } from 'next/navigation';
 import React from 'react';
 import { AuthState } from '../auth';
+import { Filter } from '@/components/Filter';
```

Comment:
This isn't resolving.  The `Filter` needs to be `filter`.

Module not found: Can't resolve '@/components/Filter'

Reply:
updated, looks like it works now